tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -348,3 +348,91 @@ TEST(caffe2, parallelBatchedMatmulRHS) {
   // We don't actually check that the output is correct, because this
   // should be covered in the OperatorTest for MatMul already.
 }
+
+/// Test loading clip op from a Caffe2 model.
```

Comment:
nit: Can you specify either here or elsewhere what the min/max values of the Clip are, so we wouldn't need to go to look at the pbtxt to find out?